letsencrypt_transip_nl/data/ir_config_parameter.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="transip" model="ir.config_parameter">
```

Comment:
this will conflict with other dns providers. just don't set anything there, the user has to configure this anyways